network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/app_install_hero.html

```diff
@@ -0,0 +1,49 @@
+{% load i18n wagtailcore_tags wagtailimages_tags %}
+
+{% image page.hero_background original as hero_background %}
+
+<div class="app-install-page-hero tw-py-0 tw-flex tw-flex-col tw-justify-center tw-h-auto tw-min-h-[25em] tw-bg-center tw-bg-cover medium:tw-py-24 medium:tw-h-[65vh] large:tw-h-[75vh]" style="background-image:url('{{ hero_background.url }}')">
```

Comment:
Instead of using inline `style` for background image, let's use Tailwind class instead.